.github/workflows/ruby_build.yml

```diff
@@ -0,0 +1,30 @@
+name: Ruby Build
+
+on:
+  release:
+    types: published
```

Comment:
Does this also happen for drafts?

Reply:
Not drafts, but does work for pre-releases. I kind of assumed the whole point of a draft release was that it wasn't ready yet and thus you wouldn't want to build it. I don't tend to use them myself though, so maybe I'm missing a use case.

Reply:
No sorry I meant pre-release!